posthog/batch_exports/models.py

```diff
@@ -64,6 +64,8 @@ class Status(models.TextChoices):
     batch_export = models.ForeignKey(
         "BatchExport", on_delete=models.CASCADE, help_text="The BatchExport this run belongs to."
     )
+    workflow_id = models.TextField(null=True, help_text="The Workflow id as set by Temporal.")
```

Comment:
There's something that irks me having the temporal implementation bleed over into this part of the data model but I don't have a better solution.

Reply:
I removed this.

Reply:
I was concerned about the existing export runs not being reset-able as they don't have these ID fields. So, I removed the ID fields and do resets without them. Turns out this is viable as:

1. Temporal can handle resets with only a Workflow ID (which we already know the UUID part as it's the BatchExport.id field).
2. We already know the data_interval_end to construct the final part of the Workflow ID.